portal/app/[locale]/_components/navbar/_components/socialLinks.tsx

```diff
@@ -3,41 +3,69 @@ import { ExternalLink } from 'components/externalLink'
 import { DiscordIcon } from 'components/icons/discordIcon'
 import { GithubIcon } from 'components/icons/githubIcon'
 import { LinkedinIcon } from 'components/icons/linkedinIcon'
+import { TiktokIcon } from 'components/icons/tiktokIcon'
 import { TwitterIcon } from 'components/icons/twitterIcon'
 import { YoutubeIcon } from 'components/icons/youtubeIcon'
 import hemiSocials from 'hemi-socials'
 import { useUmami } from 'hooks/useUmami'
+import dynamic from 'next/dynamic'
 import { Suspense } from 'react'
 
-const { discordUrl, githubUrl, linkedinUrl, twitterUrl, youtubeUrl } =
-  hemiSocials
+import { TelegramButton } from './telegram/telegramButton'
+
+const {
+  discordUrl,
+  githubUrl,
+  linkedinUrl,
+  telegramCommunityUrl,
+  telegramNewsUrl,
+  tiktokUrl,
+  twitterUrl,
+  youtubeUrl,
+} = hemiSocials
 
 const svgCss = '[&_path]:hover:fill-black'
 
+const Telegram = dynamic(() => import('./telegram').then(mod => mod.Telegram), {
+  loading: () => <TelegramButton isOpen={false} />,
+  ssr: false,
+})
+
 const UI = ({
   addTracking,
 }: {
   addTracking?: (event: AnalyticsEvent) => () => void | undefined
 }) => (
   <div
-    className="mb-3 mt-0 flex flex-wrap items-center justify-center gap-x-6
-      md:mb-6 md:mt-6 md:h-10 md:gap-x-4 md:border-t md:border-neutral-300/55 md:py-4 lg:mb-0"
+    className="mb-3 mt-0 flex flex-wrap items-center justify-between gap-x-6 overflow-visible
+      px-3 md:mb-6 md:mt-6 md:h-10 md:gap-x-2 md:border-t md:border-neutral-300/55 md:!px-0 md:py-2 lg:mb-0"
   >
     <ExternalLink href={twitterUrl} onClick={addTracking?.('nav - x')}>
       <TwitterIcon className={svgCss} />
     </ExternalLink>
     <ExternalLink href={discordUrl} onClick={addTracking?.('nav - discord')}>
       <DiscordIcon className={svgCss} />
     </ExternalLink>
-    <ExternalLink href={githubUrl} onClick={addTracking?.('nav - gitHub')}>
-      <GithubIcon className={svgCss} />
-    </ExternalLink>
+    <Telegram
+      telegramCommunityUrl={telegramCommunityUrl}
+      telegramNewsUrl={telegramNewsUrl}
+    />
     <ExternalLink href={linkedinUrl} onClick={addTracking?.('nav - linkedIn')}>
       <LinkedinIcon className={svgCss} />
     </ExternalLink>
+    <ExternalLink href={githubUrl} onClick={addTracking?.('nav - gitHub')}>
+      <GithubIcon className={svgCss} />
+    </ExternalLink>
     <ExternalLink href={youtubeUrl} onClick={addTracking?.('nav - youtube')}>
       <YoutubeIcon className={svgCss} />
     </ExternalLink>
+    <ExternalLink
+      className="-ml-1"
```

Comment:
This causes the icons to be evenly distributed by just moving the last one 

```suggestion
```

This becomes more noticeable if you add a border to each icon

<img width="280" height="86" alt="Image" src="https://github.com/user-attachments/assets/42dbf4f3-44a9-4804-91f3-07de7cd78214" />